deployment/hasura/migrations/Aerie/3_sequence_parcel/down.sql

```diff
@@ -0,0 +1,203 @@
+
+------------------------------------
+--- Modify Expanded Sequences Table
+------------------------------------
+alter table sequencing.expanded_sequences
+  add column edsl_string text;
+
+update sequencing.expanded_sequences
+set edsl_string = '{}'
+where edsl_string is null;
+
+alter table sequencing.expanded_sequences
+  alter column edsl_string set not null;
```

Comment:
Nonblocking: this part of the migration could be simplified slightly by temporarily giving the column a `default` value:
```sql
------------------------------------
--- Modify Expanded Sequences Table
------------------------------------
alter table sequencing.expanded_sequences
  add column edsl_string text not null default '{}';

alter table sequencing.expanded_sequences
  alter column edsl_string drop default;
```

Reply:
Cool, I made the change.